.github/kind-cluster-1.23.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Cluster
```

Comment:
I don't think you need to include 1.23 yet, as this commit's purpose is only to test e2e. Hence, it doesn't add support of a new version. 

If you want to try supporting 1.23, I suggest you to split that into its own commit.

Reply:
done !